.github/workflow/push_mirror.yaml

```diff
@@ -0,0 +1,31 @@
+name: Mirroring
+
+# triggers github actions everytime there is a push or mr
+on: [push, delete]
+
+jobs:
+  # To test on HPC resources must mirror the repo and trigger a pipeline
+  to_pnnl_gitlab:
+     # Latest Ubuntu as of Feb 2023 is 20.04
+     runs-on: ubuntu-latest
+     steps:
+       # Action checks-out your repository under $GITHUB_WORKSPACE, so your workflow can access it.
+       - uses: actions/checkout@v1
+       #  Action for mirroring your commits to a different remote repository
+       - uses: yesolutions/mirror-action@master
+       # Give actions access to some secrets
+       with:
+         REMOTE: ${{ secrets.GIT_REPO_URL}}
+         GIT-USERNAME:  ${{ secrets.GIT_USER}}
+         GIT-PASSWORD:  ${{ secrets.GIT_PASSWORD}}
+
+       - name: Trigger Pipeline
+         run: |
+          # TODO define PNNL_PIPELINE_TRIGGER_TOKEN
+          # Create a trigger token and replace the url
+          response=$(curl -X POST -F token=${{ secrets.PNNL_PIPELINE_TRIGGER_TOKEN }} -F ref=${BRANCH_NAME} https://gitlab.pnnl.gov/api/v4/projects/769/trigger/pipeline)
+          exit_code=$?
+          sudo apt install jq
+          pipeline_id=$(echo $response | jq '.id' | sed 's/"//g')
+          echo "PIPELINE_ID=${pipeline_id}" >> $GITHUB_ENV
+          exit $exit_code
```

Comment:
```suggestion
```
None of this is necessary as it doesn't change the result of the pipeline, and we don't need the pipeline_id in this context